tests/Integration/Tasks/NetworkInventoryTest.php

```diff
@@ -311,7 +289,7 @@ public function prepareDb()
         $ipranges_id = $pfIPRange->add($input);
         $this->assertNotFalse($ipranges_id);
 
-       // Allow all agents to do network inventory
+       // Allow all agents to do network discovery
```

Comment:
Seems it's doing inventory here, not discovery

Reply:
done